docs/app/docs/page/blocks/basic-blocks/ngd-overview-block.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'ngd-overview-block',
+  template: `
+    <div class="block-container">
+      <h2 class="class-name">
+        <a [routerLink]="" fragment="{{source.name}}" ngdFragment></a>
+        {{source?.name}}
+      </h2>
+      <ng-container class="description" *ngFor="let node of overview">
+        <p *ngIf="node.type === 'text'" ngdDescription class="description">
+          {{node.content}}
+        </p>
+        <ngd-live-example *ngIf="node.type === 'live-example'" [id]="node.content">
```

Comment:
move to types?

Reply:
this type exists in parser model, but I don't think we have to use them now. It would be smarter to integrate all types from parser model at one time.